src/browser/main.coffee

```diff
@@ -69,7 +69,9 @@ setupCrashReporter = ->
 setupAtomHome = ->
   return if process.env.ATOM_HOME
 
-  atomHome = path.join(app.getHomeDir(), '.atom')
+  atomHome = path.join(path.dirname(process.execPath), 'userdata')
```

Comment:
So on Windows when you use the installer, the `proces.execPath` will be something like `C:\Users\kevin\AppData\Local\atom\app-0.191.0\atom.exe` so if you put a `userdata` folder in `C:\Users\kevin\AppData\Local\atom\app-0.191.0\` it will also be deleted the next time Atom automatically updates.

If this is something that will be supported by the app, the location needs to be someplace that isn't deleted when Atom updates so the person doesn't lose their packages, config, etc.